migrations/20170906143027_telemetry.js

```diff
@@ -0,0 +1,13 @@
+exports.up = function (knex) {
+  return knex.schema.createTable('telemetry', (t) => {
+    t.float('latitude');
+    t.float('longitude');
+    t.integer('charge');
+    t.integer('drone_id').unsigned().primary();
+    t.foreign('drone_id').references('drone.id');
+    t.text('status');
```

Comment:
@pegaheinaliyan Can you check ENUM for the status column? If you can use ENUM, you can define what status can be added in the column.